### PR TITLE
clang-format

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
Retest this please

**Comment by eile** [comment]
Retest this please

File: zeroeq/connection/broker.h [hunk 1]
```diff
@@ -89,11 +92,13 @@ class Broker : public Receiver
...
     // Receiver API
-    void addSockets( std::vector< zeroeq::detail::Socket >& entries ) final;
-    void process( zeroeq::detail::Socket& socket, uint32_t timeout ) final;
-    void addConnection( const std::string& ) final { ZEROEQDONTCALL; } // LCOV_EXCL_LINE
+    void addSockets(std::vector<zeroeq::detail::Socket>& entries) final;
+    void process(zeroeq::detail::Socket& socket, uint32_t timeout) final;
+    void addConnection(const std::string&) final
+    {
+        ZEROEQDONTCALL;
+    } // LCOV_EXCL_LINE
```
Comment (rdumusc) [inline zeroeq/connection/broker.h]: still works when not inline?

File: zeroeq/detail/sender.h [hunk 2]
```diff
@@ -6,82 +6,78 @@
 #ifndef ZEROEQ_DETAIL_SENDER_H
 #define ZEROEQ_DETAIL_SENDER_H
 
+#include <stdexcept>
```
Comment (rdumusc) [inline zeroeq/detail/sender.h]: move after <zeroeq> includes?

**Comment by eile** [comment]
json-cpp excluded

File: zeroeq/detail/sender.h (hunk 2 above)
Comment (eile) [inline zeroeq/detail/sender.h]: done

File: zeroeq/connection/broker.h (hunk 1 above)
Comment (eile) [inline zeroeq/connection/broker.h]: fixed

File: zeroeq/connection/broker.h [hunk 3]
```diff
@@ -89,11 +92,10 @@ class Broker : public Receiver
     detail::Broker* const _impl;
 
     // Receiver API
-    void addSockets( std::vector< zeroeq::detail::Socket >& entries ) final;
-    void process( zeroeq::detail::Socket& socket, uint32_t timeout ) final;
-    void addConnection( const std::string& ) final { ZEROEQDONTCALL; } // LCOV_EXCL_LINE
+    void addSockets(std::vector<zeroeq::detail::Socket>& entries) final;
+    void process(zeroeq::detail::Socket& socket, uint32_t timeout) final;
+    void addConnection(const std::string&) final { ZEROEQDONTCALL; }
```
Comment (tribal-tec) [inline zeroeq/connection/broker.h]: LCOV_EXCL_LINE is imo still needed here

File: zeroeq/http/server.h [hunk 4]
```diff
@@ -258,28 +263,31 @@ class Server : public zeroeq::Receiver
...
     // Receiver API
-    void addSockets( std::vector< detail::Socket >& entries ) final;
-    void process( detail::Socket& socket, uint32_t timeout ) final;
-    void addConnection( const std::string& ) final { ZEROEQDONTCALL; } // LCOV_EXCL_LINE
+    void addSockets(std::vector<detail::Socket>& entries) final;
+    void process(detail::Socket& socket, uint32_t timeout) final;
+    void addConnection(const std::string&) final
+    {
+        ZEROEQDONTCALL;
+    } // LCOV_EXCL_LINE
```
Comment (tribal-tec) [inline zeroeq/http/server.h]: this works?

File: zeroeq/connection/broker.h (hunk 3 above)
Comment (rdumusc) [inline zeroeq/connection/broker.h]: you could use LCOV_EXCL_START + LCOV_EXCL_STOP

**Comment by eile** [comment]
What an effort for something nobody looks at. :-/

File: zeroeq/http/server.h (hunk 4 above)
Comment (eile) [inline zeroeq/http/server.h]: No: It's either remove (as above) or have at least four lines with two LCOV comments. I vote remove.

Comment (tribal-tec) [inline zeroeq/http/server.h]: you get a false positive in the coverage report then. zeroeq had almost 100%, so then we can also disable coverage report generation and save 200 lines of cmake code

Comment (hernando) [inline zeroeq/http/server.h]: I don't get this. Can't it be

    void addConnection(const std::string&) final
    {
        ZEROEQDONTCALL; // LCOV_EXCL_LINE
    }

I don't see anything wrong with that.

Comment (eile) [inline zeroeq/http/server.h]: This will still have no coverage on line 1 and 4.

**Comment by rdumusc** [comment]
Despite your comment, jsoncpp is still reformatted. Other than that, +1.

**Comment by eile** [comment]
@rdumusc : this one is a bitch. Will do a third try to revert it. grmbl.

**Comment by eile** [comment]
Ping! I still don't have an approval!

**Pull request merged**